packages/components/src/components/list/List.d.ts

```diff
@@ -0,0 +1,44 @@
+declare module '@enykeev/react-virtualized/dist/es/List' {
+    import { ListProps } from '@enykeev/react-virtualized/dist/es/List';
+    import type { Grid } from 'react-virtualized';
+
+    type TRowRenderer = {
+        style: CSSProperties;
+        index: number;
+        key: string;
+        parent: React.RefObject<Grid>;
+    };
+
+    type TList = {
+        autoHeight: boolean;
+        className?: string;
+        deferredMeasurementCache: any;
+        height: number;
+        overscanRowCount: number;
+        ref: any;
```

Comment:
@niloofar-deriv it doesn't seem like List accepts ref according to the library's docs: https://github.com/bvaughn/react-virtualized/blob/master/docs/List.md